bin/miq_gem_push.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'manageiq/release/settings'
+
+require 'aws-sdk-s3'
+require 'fileutils'
+require 'rubygems'
+require 'rubygems/package'
+require 'zlib'
+
+gems_to_push = ARGV
+raise "Please specify at least one gem" if gems_to_push.empty?
+
+client = Aws::S3::Client.new(
+  :access_key_id     => Settings.manageiq_rubygems.s3_access_key,
+  :secret_access_key => Settings.manageiq_rubygems.s3_secret_key,
```

Comment:
not for this PR, but would be nice to pull secret key from an ENV var first if given, as I'd rather not keep my key in a conf file.

Reply:
something like `:secret_access_key => ENV.fetch("AWS_SECRET_ACCESS_KEY", Settings.manageiq_rubygems.s3_secret_key)`

Reply:
Same with the access_key_id...something like `ENV.fetch("AWS_ACCESS_KEY_ID", Settings.manageiq_rubygems.s3_access_key)`